src/index.js

```diff
@@ -99,9 +99,28 @@ const init = ({
 
 const createId = init();
 
+const isCuid = (id, { minLength = 2, maxLength = bigLength } = {}) => {
+  const length = id.length;
+  const regex = /^[0-9a-z]+$/;
```

Comment:
Isn't first character just [a-z] ... should be `/^[a-z][0-9a-z]+$/` as per me

<img width="299" alt="Screenshot 2023-02-21 at 4 20 15 PM" src="https://user-images.githubusercontent.com/4970624/220324961-4e3552a4-25f9-488e-8b14-68ce13226229.png">

Reply:
This is not a bug.

Yes, that is the current structure of cuids. However, you should treat the structure of the id as a black-box from a validation/type checking perspective. If you ever decide you need to switch id solutions, you will need to do extra work if you enforce the id specification too strictly.

Reply:
@ericelliott - In that case how about having something like `initialCharAsLetter` as option

```javascript 
isCuid(id, { initialCharAsLetter: true })
```

And if it's true we check for whether initial character is a letter or not?

Reply:
No, you should never do the initial letter check for validation, for the reasons described above.

Reply:
Ohk if you say so @ericelliott ... thanks!